etsi_its_conversion/etsi_its_conversion/include/etsi_its_conversion/Converter.hpp

```diff
@@ -99,14 +99,23 @@ class Converter : public rclcpp::Node {
     template <typename T_ros, typename T_struct>
     bool encodeRosMessageToUdpPacketMessage(const T_ros& msg, UdpPacket& udp_msg, const asn_TYPE_descriptor_t* type_descriptor, std::function<void(const T_ros&, T_struct&)> conversion_fn, const int btp_header_destination_port) const;
 
+    template <typename T_ros, typename T_struct, typename T_request, typename T_response>
+    void rosToUdpSrvCallback(const std::shared_ptr<T_request> request,
+                            std::shared_ptr<T_response> response,
+                            const std::string& type,
+                            const asn_TYPE_descriptor_t* asn_type_descriptor,
+                            std::function<void(const T_ros&, T_struct&)> conversion_fn);
```

Comment:
Can probably use one less template type like this; same for the other callback

```suggestion
    template <typename T_ros, typename T_struct, typename T_srv>
    void rosToUdpSrvCallback(const std::shared_ptr<T_srv::Request> request,
                            std::shared_ptr<T_srv::Response> response,
                            const std::string& type,
                            const asn_TYPE_descriptor_t* asn_type_descriptor,
                            std::function<void(const T_ros&, T_struct&)> conversion_fn);
```